src/Sylius/Bundle/ResourceBundle/spec/Controller/ParametersParserSpec.php

```diff
@@ -25,165 +24,119 @@
  */
 final class ParametersParserSpec extends ObjectBehavior
 {
-    function let(ContainerInterface $container, ExpressionLanguage $expression)
```

Comment:
Why did you remove this method? 
```php
function let()
{
        $this->beConstructedWith(new Container(), new ExpressionLanguage());
}
```

And call `$this->beConstructedWith($container, $expression);` when is needed